Sources/Sentry/include/SentrySystemWrapper+Tests.h

```diff
@@ -0,0 +1,8 @@
+#import "SentrySystemWrapper.h"
+
+@interface
```

Comment:
`m`: This file could be under `Tests/SentryTests` and not `Sources/...`. Same applies for `SentryNSTimeWrapper+Tests.h`